libs/core/algorithms/include/hpx/parallel/datapar/transform_loop.hpp

```diff
@@ -11,6 +11,7 @@
 #if defined(HPX_HAVE_DATAPAR)
 #include <hpx/datastructures/tuple.hpp>
 #include <hpx/execution/traits/is_execution_policy.hpp>
+#include <hpx/executors/datapar/execution_policy.hpp>
```

Comment:
I think `hpx/parallel/datapar/transfer.hpp` should also have this header included. It doesn't show up on the header tests because it includes `hpx/parallel/datapar/transform_loop.hpp` but it uses `simd_policy` directly so it should have the include as well.